package.json

```diff
@@ -100,7 +100,7 @@
                     "django"
                 ],
                 "filenamePatterns": [
-                    "**/templates/**/*"
+                    "**/templates/**/*.txt"
```

Comment:
`"**/templates/**/*.{txt, tex}"`

Something like this is possible, but as you said I am unsure on what it should be limited to. 

I am not well versed in vscode extensions, but perhaps this can be made configurable?